README.md

```diff
@@ -36,7 +36,7 @@ The web project (written in Node.js) mimics a mock-up of a web music player and
 mentioned services. It will display the currently played song title and artist name together with
 the cover art. Moreover, it will also show the current top songs of the artist.
 Replace the URLs with your service endpoints inside `routes/index.js` of the web project if necessary.
-The user interface is available at `http://{docker-machine-ip}:3000`.
+The user interface is available at `http://{docker-machine-ip}:8080`.
```

Comment:
Der Port ist in diesem Repo noch 3000:
https://github.com/stefan-kolb/docker-dojo/blob/709136b125b4ecd0e93981fc3a70ed83c29d0d74/web/bin/www#L15